.env.development

```diff
@@ -46,6 +46,7 @@ FEATURE_ENABLE_AUTO_APPLIED_LICENSES='true'
 LEARNING_TYPE_FACET='true'
 LEARNER_SUPPORT_URL='https://support.edx.org/hc/en-us'
 LEARNER_SUPPORT_SPEND_ENROLLMENT_LIMITS_URL='http://edx.org'
+LEARNER_SUPPORT_ABOUT_DEACTIVATION_URL='http://edx.org'
```

Comment:
Related PR to add this config setting to stage/prod: https://github.com/edx/edx-internal/pull/8818